src/bp-core/bp-core-filters.php

```diff
@@ -70,6 +70,10 @@
 add_filter( 'bp_core_widget_user_display_name', 'strip_tags' );
 add_filter( 'bp_core_widget_user_display_name', 'esc_html' );
 
+// Redirects.
+add_filter( 'login_redirect', 'bb_login_redirect', PHP_INT_MAX, 3 );
```

Comment:
remove this hook as we already have  `add_filter( 'login_redirect', 'bp_login_redirect', 10, 3 )`;